### ISSUE TITLE
Unexpected type conversion

### ISSUE THREAD

**Issue by rpdprd** [typicode/json-server#365]
Is there a way for the JSON document to be stored as is? For example, string "true" and string "42" are converted to boolean ant number respectively.

```
curl -H "Content-Type: application/json" -X POST -d '{"field1":"xyz","field2":"true","field3":"42","field4":"null"}' http://localhost:3000/example
{
  "field1": "xyz",
  "field2": true,
  "field3": 42,
  "field4": "null",
  "id": 1
}
```

While it may not be significant in javascript, the resulting document cannot be easily decoded to a map in Go (and maybe other languages) if the expected type is a string.

Thanks

**Comment by typicode** [comment]
Hi @rpdprd,

Not right now, but it's something that I'm working on (`next` branch).

**Comment by typicode** [comment]
I've published `v0.9.0-beta.2` which should improve this. Auto convert has been removed.

If you want, you can give it a try using `npm install json-server@next`.
`README` for `v0.9.0` can be found in the `next` branch https://github.com/typicode/json-server/tree/next

Feedback is welcome :)

**Comment by typicode** [comment]
Should be good in `v0.9.0`
